src/cli/node/with_node.rs

```diff
@@ -145,8 +153,8 @@ pub fn exec_cmd(command: Command, mut config: Config) -> Result<(), failure::Err
             let write_to_path = match (write, write_to) {
                 // Don't write
                 (false, None) => None,
-                // Write to default storage path
-                (true, None) => Some(config.storage.db_path),
+                // Write to the same folder where the config file is located
+                (true, None) => config_path.and_then(|path| path.parent().map(Path::to_path_buf)),
```

Comment:
If `config_path` is `None` this silently ignores the `--write` flag. We should return an error instead.

Reply:
Sounds fair to me.

Reply:
@tmpolaczyk commit 3f671b61e037e6d3cbb0f3415107dfd68f82acdf implements your suggestion